src/FunctionLoader.ts

```diff
@@ -70,6 +70,19 @@ export class FunctionLoader implements IFunctionLoader {
             throw new InternalException(`Function code for '${functionId}' is not loaded and cannot be invoked.`);
         }
     }
+
+    isUsingMjs(filePath: string, packageJson: Object): Boolean {
+        if (filePath.endsWith('.mjs')) {
+            return true;
+        }
+        if (filePath.endsWith('.cjs')) {
+            return false;
+        }
+        if (packageJson['type'] && packageJson['type'] === 'module') {
+            return true;

```

Comment:
nit: How about we rename this to `isESModule`? This case returns true even though the file name isn't "mjs"

Reply:
Done